phpBB/config/default/routing/routing.yml

```diff
@@ -8,6 +8,14 @@
 # instantiate the "foo_service" service and call the "method" method.
 #
 
+phpbb_feed_routing:
+    resource: "feed.yml"
+    prefix: /feed
+
+phpbb_feed_index:
```

Comment:
no because the goal is to access it without the leading slash

It is possible to support both with a single route definition but it requires more work and can be tricky for BC